### PR TITLE
(chore) - save bytes by removing the string from the production build

### PR DESCRIPTION
**Pull request by JoviDeCroock**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export const createContext = (defaultValue) => {
 export const useContextSelector = (context, selector) => {
   const listeners = context[CONTEXT_LISTENERS];
   if (!listeners) {
-    throw new Error('useContextSelector requires special context');
+    throw new Error(process.env.NODE_ENV !== 'production' ? 'useContextSelector requires special context' : '');
```
Comment (dai-shi) [inline src/index.js]: If we were to go more, how about this?
```suggestion
    throw (process.env.NODE_ENV !== 'production' ? new Error('useContextSelector requires special context') : new Error());
```
Would this save two bytes?

**Comment by dai-shi** [comment]
Do you think we can apply the same technique for a symbol?
```js
const CONTEXT_LISTENERS = process.env.NODE_ENV !== 'production' ? Symbol('CONTEXT_LISTENERS') : Symbol();
```

**Comment by JoviDeCroock** [comment]
> Do you think we can apply the same technique for a symbol?

This should work as far as I am aware

**Comment by dai-shi** [comment]
Would you try that too in this PR? I'm not sure if eslint rules complain.

**Comment by JoviDeCroock** [comment]
ESLint breaks like this yes

**Comment by dai-shi** [comment]
https://eslint.org/docs/rules/symbol-description

> this rule can be safely turned off if you don’t want to enforce presence of description when creating Symbols.

Well, let us turn it of then...

**Comment by dai-shi** [comment]
I will take it over. Thanks!

**Comment by dai-shi** [comment]
https://www.npmjs.com/package/use-context-selector/v/1.0.1

**Pull request merged**